content/docs/integrations/istio/ambient-ingress.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Ingress to ambient mesh
+weight: 30
+description: Use kgateway as the ingress gateway for your ambient mesh. 
+---
+
+Use kgateway as the ingress gateway for your ambient mesh. 
+
+## About ambient mesh
+
+Solo collaborated with Google to develop ambient mesh, a new “sidecarless” architecture for the Istio service mesh. Ambient mesh uses node-level ztunnels to route and secure Layer 4 traffic between pods with mutual TLS (mTLS). Waypoint proxies enforce Layer 7 traffic policies whenever needed. To onboard apps into the ambient mesh, you simply label the namespace the app belongs to. Because no sidecars need to be injected in to your apps, ambient mesh significantly reduces the complexity of adopting a service mesh. 
```

Comment:
should this be something like "Solo.io and Google collaborated..." to make it more generic